draft-ietf-lamps-csr-attestation.md

```diff
@@ -631,6 +629,63 @@ Columns:
 -  Description: Begins with id-ata
 -  References: RFC or other document
 
+## Attestation Evidence OID Registry
+
+IANA is asked to create a registry that helps developers to find
+OID/Evidence mappings.
+
+Registration requests are evaluated using the criteria described in
+the registration template below after a three-week review period on
+the [[TBD]] mailing list, on the advice of one or more Designated
+Experts {{RFC8126}}.  However, to allow for the allocation of values
```

Comment:
```suggestion
the registration template below after a three-week review period on
the [[TBD]] mailing list, with the advice of one or more Designated
Experts {{RFC8126}}.  However, to allow for the allocation of values
```

I think that it's a three-week review,, *AND* advice of DEs.  With the previous wording, I could interpret "or"